artiq/coredevice/trf372017.py

```diff
@@ -4,20 +4,20 @@ class TRF372017:
     For possible values, documentation, and explanation, see the datasheet.
     https://www.ti.com/lit/gpn/trf372017
     """
-    rdiv = 21  # 13b
+    rdiv = 2  # 13b - highest valid f_PFD
     ref_inv = 0
     neg_vco = 1
     icp = 0  # 1.94 mA, 5b
     icp_double = 0
-    cal_clk_sel = 12  # /16, 4b
+    cal_clk_sel = 0b1110  # div64, 4b
 
-    ndiv = 420  # 16b
-    pll_div_sel = 0  # /1, 2b
-    prsc_sel = 1  # 8/9
+    nint = 23  # 16b - lowest value suitable for fractional & integer mode
```

Comment:
What is the LO frequency with this? That would be good to document.
Do you approximate the previous default frequency or is there a reason to change that?

Reply:
This does not approximate the previous frequency. With these setting the default VCO frequency is 2.875 GHz. This was selected as it is the lowest VCO frequency allowing both fractional and integer mode. These values are likely to be reconfigured when used in an experiment. To me, the most sensible default is the one requiring the least modifications in test-setups.

Reply:
Ah. But the divider was 2. So the LO was at ~1.4 GHz (https://github.com/quartiq/phaser/issues/3).
Then those values are fine with me. Can you add a line saying what frequency the LO and VCO are at with this?

Reply:
Sure